packages/ui/src/forum/components/threads/ThreadListItem.tsx

```diff
@@ -39,7 +39,7 @@ export const ThreadListItem = ({ thread, isArchive }: ThreadListItemProps) => {
         </Thread>
       </Tooltip>
 
-      <TextMedium bold>{thread.visiblePostsCount - 1}</TextMedium>
+      <TextMedium bold>{ thread.visiblePostsCount > 0 ? thread.visiblePostsCount - 1 : thread.visiblePostsCount }</TextMedium>
```

Comment:
slightly better style (without understanding the [decrement in the original](https://github.com/Joystream/pioneer/commit/6083e0563875fe7362a957e46d0f2e750880dab6#diff-00fc2f0ba19bebbf9a9fe6492f3c2dc5c98fdbf289f60838e9fcc811f6313808R31))
>      <TextMedium bold>{thread.visiblePostsCount > 1 ? thread.visiblePostsCount - 1 : 0}</TextMedium>

Reply:
Agree, using 0 is cleaner.
```suggestion
      <TextMedium bold>{ thread.visiblePostsCount > 0 ? thread.visiblePostsCount - 1 : 0 }</TextMedium>
```

Reply:
accepted suggestion and changed accordingly.